spec/spec_helper.rb

```diff
@@ -16,7 +16,7 @@
 Dir[File.join(File.dirname(__FILE__), 'support/**/*.rb')].each { |f| require f }
 
 # Requires factories defined in lib/solidus_volume_pricing/factories.rb
```

Comment:
Do we want to update this comment as well? Factories should be at `lib/solidus_volume_pricing/testing_support/factories.rb` now, isn't it?